JakinChan200/CS180_Roald
# Countries endpoint and category filtering

## Overview
Added a collection endpoint for all countries, specific endpoints for each country, and the ability to filter within a country by category.


## Materials
- [Issue] #18 

## Pictures
![Countries-Endpoint-PR](https://user-images.githubusercontent.com/24843819/163265174-f3da2ea5-216b-4c81-b65a-e7569b2ad487.gif)

## Manual Testing Guide
- Changes are at https://localhost:8000/api/countries and https://localhost:8000/api/countries/[country]
- See GIF above demonstrating changes

Pull requests should receive approval before merge.
